docs/CONFIGURATION.md

```diff
@@ -0,0 +1,270 @@
+## For English press [here](#configuration)
+# Konfiguracja
+
+Ten dokument zawiera przegląd wartości konfigurowalnych w pliku [`values.yaml`](../charts/kam/values.yaml) dla chartu Helm. Te wartości mogą być dostosowywane w celu skonfigurowania wdrożenia aplikacji. W celu uzyskania szczegółowych informacji jak wdrożyć aplikację, odwołaj się do pliku [DEPLOYMENT.md](./DEPLOYMENT.md).
+
+## Ogólna konfiguracja
+
+- **nameOverride**: Nadpisuje nazwę chartu. Przydatne, jeśli chcesz wdrożyć wiele instancji chartu z różnymi nazwami.
+- **fullnameOverride**: Całkowicie nadpisuje wygenerowaną nazwę chartu.
+- **podAnnotations**: Adnotacje do dodania do zasobów pod.
+- **podLabels**: Etykiety do dodania do zasobów pod.
+
+## Globalna konfiguracja
+
+### **global.env.FRONTEND_KEYCLOAK_URL**
+- **Opis**: URL do serwera Keycloak używanego przez frontend. Jest wykorzystywany przez frontend do celów uwierzytelniania. Powinien być publicznie dostępny.
+- **Wymagane**: Tak
```

Comment:
Well, it's my bad too, but it's not mandatory, when KEYCLOAK_LOGIN_URL, KEYCLOAK_LOGOUT_URL and KEYCLOAK_TOKEN_URL are set